src/main/java/codechicken/multipart/api/ItemMultipart.java

```diff
@@ -24,6 +27,10 @@ public ItemMultipart(Properties properties) {
     @Nullable
     public abstract MultiPart newPart(MultipartPlaceContext context);
 
+    protected List<MultiPart> newParts(MultipartPlaceContext context) {
```

Comment:
Should we use `Collection` here instead of list? Do we need to enforce an order?

Reply:
Done. Will also change `TileMultipart.canPlaceParts` to intake `Iterable<MultiPart>` instead of list to match everything else.